dlt/helpers/ibis.py

```diff
@@ -53,12 +60,18 @@ def create_ibis_backend(
 ) -> BaseBackend:
     """Create a given ibis backend for a destination client and dataset"""
 
+    # ensure destination is a Destination instance
+    if not isinstance(destination, Destination):
+        destination = Destination.from_reference(destination)
+
     # check if destination is supported
-    destination_type = Destination.from_reference(destination).destination_type
-    if destination_type not in SUPPORTED_DESTINATIONS:
+    if not any(issubclass(destination.spec, spec_class) for spec_class in SUPPORTED_DESTINATIONS):
```

Comment:
AFAK `issubclass` takes tuple of classes to check. so `any` wont be needed. pls check

Reply:
also you could just raise in the last `else`, after all the possibilities are expired? in that case you may not need `SUPPORTED_DESTINATIONS` at all

Reply:
I think it would be cool to save the sqlglot dialect to be used in the destination capabilities, this will be very useful for the transformations stuff I am doing because I need to know which dialect to select there too. The default dialect in the generic_capabilities can be duckdb. Or is there a reason we should not have this info there?

Reply:
let's do separate ticket